gems/pending/appliance_console/cli.rb

```diff
@@ -48,13 +52,25 @@ def database?
     end
 
     def local_database?
-      hostname && local?(hostname)
+      database? && local?(hostname)
     end
 
     def certs?
       options[:postgres_client_cert] || options[:postgres_server_cert] || options[:http_cert]
     end
 
+    def uninstall_ipa?
+      options[:uninstall_ipa]
```

Comment:
this is set to true when the user specified `--uninstall-ipa`
So it is true, then the user asked for this action to be performed.

We want to educate when the user doesn't ask for any actions